packages/@aws-cdk/aws-appmesh/lib/gateway-route-spec.ts

```diff
@@ -202,9 +453,17 @@ class GrpcGatewayRouteSpec extends GatewayRouteSpec {
               virtualServiceName: this.routeTarget.virtualServiceName,
             },
           },
+          rewrite: this.match.rewriteRequestHostname === false
+            ? {
+              hostname: {
+                defaultTargetHostname: 'DISABLED',
+              },
+            }: undefined,
```

Comment:
```suggestion
          rewrite: this.match.rewriteRequestHostname === undefined
            ? undefined
            : {
              hostname: {
                defaultTargetHostname: this.match.rewriteRequestHostname ? 'ENABLED' : DISABLED',
              },
            },
```

Reply:
May I leave it as it is? Since the default for `defaultTargetHostname` is `ENABLED`, I think it is better to only specify `DISABLED`.

Reply:
Why though? What's the advantage of that?

Reply:
I would still do it, I don't see much advantage in removing the `'ENABLED'` option.

Reply:
I was under the impression that it is better to leave CloudFormation template small.
By omitting assigning `ENABLED` to `defaultTargetHostName`, `HostName` property in the CloudFormation template can be taken out.

Reply:
It's good to keep the small if the customer didn't set a property. If they set it, even though it's the default, it's fine to reflect it in the template (otherwise, they might actually be worried that their code change was ignored).

For simplicity, I would just set it if `rewriteRequestHostname` is not `undefined`.

Reply:
Thank you very much for additional detail! Will changed this accordingly.